test/scripts/sticky/test-sticky-provider-routes.sh

```
@@ -0,0 +1,375 @@
+                -H "Content-Type: application/json" \
+                -H "X-Olla-Session-ID: ${new_session}" \
+                -d "$body_json" \
+                "${OLLA_URL}${url_path}" 2>/dev/null)
+            local ep_div
+            ep_div=$(extract_header "$headers_file" "X-Olla-Endpoint")
+            if [[ "$ep_div" != "$ep1" ]]; then
+                seen_other=true
+                break
+            fi
+        done
+        $seen_other && pass "Turn 3 load balancing reaches multiple backends" || fail "Turn 3 all 30 attempts hit '${ep1}' only — balancer may be stuck"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)